src/pages/home/HomePage.tsx

```diff
@@ -73,6 +78,10 @@ export default class HomePage extends React.Component<IHomePageProps, IHomePageS
 
     public render()
     {
-        return (<QueryAndDownloadTabs store={this.store} />);
+        const blurb:JSX.Element|null = AppConfig.skinBlurb? <p style={{marginBottom:"20px"}} dangerouslySetInnerHTML={{__html: AppConfig.skinBlurb}}></p> : null;
+        return (<div>
+                   {blurb}
```

Comment:
blurb should be wrapped in div.  Lets not rely on configurators to wrap with div.